starter-code/pizza.js

```diff
@@ -1 +1,72 @@
 // Write your Pizza Builder JavaScript in this file.
+
+$(document).ready(function(){
+
+$(".opc").hide();
+toggleGreenpeppers();
+toggleMushrooms();
+togglePepperoni();
+toogleCrust();
+toogleSauce();
+totalPrice();
+});
+
```

Comment:
I see to much repetition along the next functions, you should create a single function and pass the different values as parameters of the function, you could reduce the code by the half